.storybook/main.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   addons: [
     '@storybook/addon-a11y',
     '@storybook/addon-links',
-    '@storybook/addon-essentials',
+    {name: '@storybook/addon-essentials', options: {backgrounds: false}},
```

Comment:
This removes the background switcher which isn't useful for us:

![image](https://user-images.githubusercontent.com/1863771/164721798-4e88a4d9-3c35-491c-8468-2bee6dff6355.png)